salsa_dancing_molecules/unit_test/test_energy.py

```diff
@@ -0,0 +1,13 @@
+"""Unittest for energy_calc.py."""
+
+import pytest
+from ..energy import get_energy, get_potential_energy, get_kinetic_energy
+from .mock_atoms import atoms
+
+
+def test_energy():
+    """Test the energy of a created atoms object."""
+    a = atoms()
+
+    assert get_energy(a) == get_kinetic_energy(a) \
```

Comment:
get_kinetic_energy() and get_potential_energy() are never tested. I think it would be a good idea to add two simple tests that show these two functions actually calculate what we expect them to calculate.